website/docs/tutorials/basic/running_your_app/3_working_directory.md

```diff
@@ -81,5 +81,12 @@ to_absolute_path('foo')    : /Users/omry/dev/hydra/foo
 to_absolute_path('/foo')   : /foo
 ```
 
-
 Working directory can be [customized](/configure_hydra/workdir.md).
+
+### Working directory and interpolation
+
+You may want a config option to depend on Hydra's automatically created working directory.
+This can be achieved through interpolation:
+```yaml title="config.yaml"
+logdir: ${hydra:run.dir}/logs
```

Comment:
hmm, this is not good. it will not work if you are doing a sweep.
you should be using logs relative to cwd.

Reply:
Arg, I didn't realize that the run dir and the working dir were different :/
I'll need to play a bit with all these different folders to get a better hang of how things are working in practice, and how the doc might make this clearer. In the meantime I'll get rid of this commit in the PR.